curriculum/challenges/english/07-scientific-computing-with-python/build-a-probability-calculator-project/probability-calculator.md

```diff
@@ -164,8 +164,8 @@ class UnitTests(unittest.TestCase):
     maxDiff = None
     def test_hat_draw_2(self):
         hat = probability_calculator.Hat(yellow=5,red=1,green=3,blue=9,test=1)
-        actual = hat.draw(20)
-        expected = ['yellow', 'yellow', 'yellow', 'yellow', 'yellow', 'red', 'green', 'green', 'green', 'blue', 'blue', 'blue', 'blue', 'blue', 'blue', 'blue', 'blue', 'blue', 'test']
+        actual = hat.draw(20).sort()
+        expected = ['yellow', 'yellow', 'yellow', 'yellow', 'yellow', 'red', 'green', 'green', 'green', 'blue', 'blue', 'blue', 'blue', 'blue', 'blue', 'blue', 'blue', 'blue', 'test'].sort()
```

Comment:
```suggestion
        actual = sorted(hat.draw(20))
        expected = sorted(['yellow', 'yellow', 'yellow', 'yellow', 'yellow', 'red', 'green', 'green', 'green', 'blue', 'blue', 'blue', 'blue', 'blue', 'blue', 'blue', 'blue', 'blue', 'test'])
```
`sort` method sorts in-place and returns `None`.

There are other tests checking what `draw` method returns. Shouldn't they as well check contents after sorting?

Reply:
It may be that other tests may work as expected because the extraction result is determined by the seed, I have never seen issues with those

in the case of this one the drawn balls can be in the same order as hat.contents or they can be randomly ordered, and it was against the freedom of implementing the output as one wants

thank you for checking on `sort`